opsdroid/connector/matrix/connector.py

```diff
@@ -83,6 +86,11 @@ def __init__(self, config, opsdroid=None):  # noqa: D107
         self.filter_id = None
         self.connection = None
         self.device_name = config.get("device_name", "opsdroid")
+        self.device_id = config.get("device_id")
+        self.store_path = config.get(
+            "store_path", str(Path(const.DEFAULT_ROOT_PATH).joinpath("matrix"))
+        )
+        self.__ignore_unverified__ = True
```

Comment:
Just a single underscore is convention for private variables. `__thing__` is generally only used for python internal methods etc.
```suggestion
        self._ignore_unverified = True
```